cluster/examples/kubernetes/ceph/object.yaml

```diff
@@ -56,6 +56,21 @@ spec:
     instances: 1
     # The affinity rules to apply to the rgw deployment or daemonset.
     placement:
+      podAntiAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 100
+            podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                  - key: app
+                    operator: In
+                    values:
+                      - rook-ceph-rgw
+              # topologyKey: */zone can be used to spread RGW across different AZ
+              # Use <topologyKey: failure-domain.beta.kubernetes.io/zone> in k8s cluster if your cluster is v1.16 or lower
+              # Use <topologyKey: topology.kubernetes.io/zone>  in k8s cluster is v1.17 or upper
+              topologyKey: kubernetes.io/hostname
```

Comment:
Nit: add similar changes to `object-openshift.yaml`

Reply:
Done. PTAL